tests/attestation_plugin/plugin/tests.c

```diff
@@ -402,6 +402,36 @@ static void* _find_claim(
     return NULL;
 }
 
+static uint32_t _hex_to_dec(uint8_t hex)
+{
+    if (hex >= '0' && hex <= '9')
+        return (uint32_t)hex - '0';
+    if (hex >= 'a' && hex <= 'f')
+        return (uint32_t)(hex - 'a') + 10;
+    if (hex >= 'A' && hex <= 'F')
+        return (uint32_t)(hex - 'A') + 10;
+    return 16;
```

Comment:
why 16?  when would this be hit?

Reply:
I took this from https://github.com/openenclave/openenclave/blob/master/common/sgx/tcbinfo.c#L134.

For any character other than [a,f] and [0,9] would have 16 returned. 16 is overflow in this scenario..

Reply:
Using strtol() in place of custom written _hex_to_dec().